tests/test_crawlers.py

```diff
@@ -17,9 +17,8 @@ class WebDirectoryCrawlerExceptionTestCase(unittest.TestCase):
 
     def test_lack_of_definition_of_get_download_url(self):
         """shall return "NotImplementedError" exception in the case of lack of definition of get_download_url method """
-        my_harvester = crawlers.WebDirectoryCrawler
         with self.assertRaises(NotImplementedError):
-            my_harvester.get_download_url("test_text", "test_text2")
+            crawlers.WebDirectoryCrawler.get_download_url("test_text", "test_text2")
```

Comment:
@aperrin66 
Is this want you mean to make it better?

Reply:
No, you need to create an instance of `WebDirectoryCrawler`: `crawler = crawlers.WebDirectoryCrawler('')` and then use it.
You should **never** use a method directly from the class if it is not a class or static method.

Reply:
Done in the next commit.